pkg/client/git.go

```diff
@@ -53,6 +53,7 @@ func Push(org, name, location string) error {
 	// TODO: use some real API instead...
 	cmd := exec.Command("/bin/sh", "-c", fmt.Sprintf(`
 cd "%s"
+rm -rf .git
```

Comment:
I do not think this is correct. That's why I implemented https://github.com/fuseml/fuseml-core/pull/40 so that the directory passed to codeset register command is treated as input source directory, and not as a git project.

In the case user still passes existing git repository as an input parameter to codeset, I think it's safer to fail rather than delete something user might not want to delete.

Reply:
OK, some correction of myself: this is actually done in tmp directory so the user data are not affected by fuseml action.

Still, I'm not sure if we should casually enable passing existing git repositories as initial code sources. Maybe "target directory is already under version control" error should be in place....

Reply:
What would be the the possible problems of passing a git repository?

Reply:
Probably nothing. It just did not feel right for user to pass a repository that was already initialized. It could mean they have a mess in directories that they use as an original input vs. directories with code that is being worked on.

But it could also be intentional I guess...

Reply:
The use case I see is: I have a project on another git and want to quickly give it a try on fuseml.

Reply:
OK, makes sense.

Reply:
Yes, I agree, the use case makes sense. I just hope we don't forget about this if we decide to reuse the same functionality to push code _directly_ from an existing location :)))